### ISSUE TITLE
False Positive: `no-empty` on empty `try-catch` catch block

### ISSUE THREAD

**Issue by sno2** [denoland/deno_lint#984]
### Lint Name

`no-empty`

### Code Snippet

```ts
try {
  new URL("/");
} catch {}
```

### Expected Result

No lint message.

### Actual Result

It says that you cannot have the catch block empty.  This is invalid because there are many cases where you do not want to have any code within the catch statement as it's fine if it fails.  In my project, I have a CSS4 selector parser and when you are matching a hyperlink element's href for the `local-link` pseudo-class you need to check if it is valid and then you can compare its value to the `window.location` but if the link is invalid then you can just safely move on which is where I have an empty `catch` block which catches this lint.

### Version
```
deno 1.17.0 (release, x86_64-pc-windows-msvc)
v8 9.7.106.15
typescript 4.5.2
```

**Comment by sno2** [comment]
Oh, ok. Thanks. I didn't think that comments would subside the lint.

**Comment by bartlomieju** [comment]
@sno2 when you run the linter there's a hint suggesting how to fix the problem:
```
❯ deno lint foo.js
(no-empty) Empty block statement
} catch {}
        ^^
    at /Users/biwanczuk/dev/deno/foo.js:3:8

    hint: Add code or comment to the empty block
    help: for further information visit https://lint.deno.land/#no-empty

```

**Comment by sno2** [comment]
> 

Yeah, completely my fault. I should have paid more attention. I figured that we hadn't handled for catch blocks when creating my lint.
